rir/src/compiler/opt/inline.cpp

```diff
@@ -120,11 +105,6 @@ class TheInliner {
                                 auto cloneSp =
                                     FrameState::Cast(copyFromFs->clone());
 
-                                // Remove the inlinee result from the
```

Comment:
Is this removed because now the framestate is before the call, so the result of the call is not on its stack?

Reply:
yes